pkg/csi/service/wcp/controller_helper.go

```diff
@@ -331,33 +331,43 @@ func getDatastoreURLFromStoragePool(ctx context.Context, spName string) (string,
 }
 
 // getAccessibleNodesFromStoragePool returns the accessibleNodes pertaining to the given StoragePool
-func getAccessibleNodesFromStoragePool(ctx context.Context, spName string) ([]string, error) {
+func getAccessibleNodesFromStoragePool(ctx context.Context, spName string) ([]string, string, error) {
```

Comment:
maybe we should change this function name now and also the comment on top. 
getAccessibleNodesFromStoragePool -> getStoragePoolInfo